user-guide/Advanced_Functionality/Swarming/SwarmingBookings.md

```diff
@@ -0,0 +1,118 @@
+---
+uid: SwarmingBookings
+---
+
+# Swarming bookings
+
+Since 10.4.4, it is possible to swarm bookings. To do so, you need the *Swarming* soft-launch option. 
+From 10.5.1 onwards, the feature is placed behind a different soft-launch option, namely *BookingSwarming*.
```

Comment:
Maybe we need to clarify that this means you can enable booking swarming without enabling the other swarming features? I think people might think they need to enable 'regular' swarming as well as the soft-launch option to have booking swarming from 10.5.1 onwards.